mlir/test/e2e/lit.site.cfg.py.in

```diff
@@ -36,13 +38,16 @@ except KeyError:
 # If rocm_agent_enumerator shows no viable GPUs, skip tests that need one,
 # because the default target will lead to compilation failures.
 config.no_AMD_GPU = False
+config.targetChip = ""
```

Comment:
Could you please incorporate the `-name` and no `-name` structure from https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/762 ?